src/main/java/loqor/ait/core/engine/impl/ShieldsCircuit.java

```diff
@@ -0,0 +1,52 @@
+package loqor.ait.core.engine.impl;
+
+import net.minecraft.block.Blocks;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.math.Direction;
+
+import loqor.ait.core.AITBlocks;
+import loqor.ait.core.engine.DurableSubSystem;
+import loqor.ait.core.engine.StructureHolder;
+import loqor.ait.core.engine.block.multi.MultiBlockStructure;
+
+public class ShieldsCircuit extends DurableSubSystem implements StructureHolder {
```

Comment:
Since this wont have its own block, it will need a related `SubSystemItem` for insertion
Look at `DEMAT_CIRCUIT ` for help